<html><h1>Validation FAILED.</h1><p>Build Time: 23 minutes 9 seconds 234 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/7109a858b3d0207b62093f4684545b859bd6c06f">7109a858b3d0207b62093f4684545b859bd6c06f</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :x:<pre><code>     [echo] java.lang.Exception: Found 3 formatting issues:
     [echo] 1: ./commerce-frontend-impl/build.gradle expected:&lt;...ind", version: "2.9.[9.1]"
     [echo] 
     [echo] 	compileOnly grou...&gt; but was:&lt;...ind", version: "2.9.[8]"
     [echo] 
     [echo] 	compileOnly grou...&gt;
     [echo] 2: ./commerce-payment-method-mercanet/build.gradle expected:&lt;...ind", version: "2.9.[9.1]"
     [echo] 	compileInclude gr...&gt; but was:&lt;...ind", version: "2.9.[8]"
     [echo] 	compileInclude gr...&gt;
     [echo] 3: ./commerce-cart-taglib/build.gradle expected:&lt;...ind", version: "2.9.[9.1]"
     [echo] 
     [echo] 	compileOnly grou...&gt; but was:&lt;...ind", version: "2.9.[8]"
     [echo] 
     [echo] 	compileOnly grou...&gt;
     [echo] 
     [echo] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:438)
     [echo] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:266)
     [echo] 
     [echo] FAILURE: Build failed with an exception.
     [echo] 
     [echo] * What went wrong:
     [echo] Execution failed for task ':checkSourceFormatting'.
     [echo] &gt; Process 'command '/opt/java/jdk1.8.0_121-x64/bin/java'' finished with non-zero exit value 1
     [echo] 
     [echo] * Try:
     [echo] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [echo] 
     [echo] * Exception is:
     [echo] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':checkSourceFormatting'.
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [echo] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
     [echo] 	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)</code></pre></li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-3.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/108//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>10 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-3.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/108//testReport">here</a>.</h5></div></html>

--- a/commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountUserRelLocalServiceImpl.java
+++ b/commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountUserRelLocalServiceImpl.java
@@ -152,7 +152,7 @@ public class CommerceAccountUserRelLocalServiceImpl
 		long commerceAccountId) {
 
 		List<CommerceAccountUserRel> commerceAccountUserRels =
-			commerceAccountUserRelPersistence.findByCommerceAccountUserId(
+			commerceAccountUserRelPersistence.findByCommerceAccountId(
 				commerceAccountId);
 
 		for (CommerceAccountUserRel commerceAccountUserRel :

